libusb/inc/usb_cdc.h

```diff
@@ -30,6 +30,7 @@
  * @{ */
 #define USB_CLASS_CDC                   0x02    /**<\brief Communicational Device class */
 #define USB_CLASS_CDC_DATA              0x0A    /**<\brief Data Interface class */
+#define USB_CLASS_VENDOR                0xFF
```

Comment:
probably these changes are not needed. please take a look

Reply:
removed